dbc2val/test/test_example_mapping/test_example_mapping.py

```diff
@@ -76,55 +75,56 @@ def test_parking_lock():
     # CurrentGear tests
     vss_mapping = dbc_mapping[park_lock_index]
     assert vss_mapping.vss_name == "Vehicle.Powertrain.Transmission.IsParkLockEngaged"
-    assert isinstance(vss_mapping.transform,dict)
+    assert isinstance(vss_mapping.transform, dict)
     assert "mapping" in vss_mapping.transform
     mapping_list = vss_mapping.transform["mapping"]
     assert len(mapping_list) == 4
     # And now just check some items
     assert mapping_list[0]["from"] == "DI_GEAR_D"
-    assert mapping_list[0]["to"] == False
+    assert mapping_list[0]["to"] is False
     assert mapping_list[1]["from"] == "DI_GEAR_P"
-    assert mapping_list[1]["to"] == True
+    assert mapping_list[1]["to"] is True
 
     # Now test some transforms
     result = vss_mapping.transform_value("DI_GEAR_D")
-    assert result == False
+    assert result is False
     result = vss_mapping.transform_value("DI_GEAR_P")
-    assert result == True
-    
+    assert result is True
+
     # For non successful mapping old value shall be kept and None returned
     result = vss_mapping.transform_value("Something undefined")
-    assert result == None
+    assert result is None
+
 
 def test_vehicle_speed():
     dbc_mapping = mapper["DI_uiSpeed"]
     assert len(dbc_mapping) == 2
-    
+
     if dbc_mapping[0].vss_name == "Vehicle.Speed":
         speed_index = 0
     else:
         speed_index = 1
     vss_mapping = dbc_mapping[speed_index]
     assert vss_mapping.vss_name == "Vehicle.Speed"
```

Comment:
same here

Reply:
I was just wondering if with a string compare it would be nicer to have is. Would be more readable. @erikbosch if you do not want to change/disagree we can merge.

Reply:
`==` seems to be preferred for strings.

See https://stackoverflow.com/questions/67489572/should-i-use-for-string-comparison

If using `is` flake8 complains:
```

erik@debian3:~/kuksa.val.feeders$ git commit --amend
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...........................................(no files to check)Skipped
Check for added large files..............................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

dbc2val/test/test_example_mapping/test_example_mapping.py:120:8: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
```